languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,66 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="ENUMERATION" name="enumeration" default="temp_off">
+          <rule>
```

Comment:
I threw this rule in the SentenceSourceChecker (since it cannot be checked with the Rule Editor) and found a few problems:

> Nous partageons donc avec nos auteurs les frais et les risques liés à la publication d'un ouvrage.

I'll send you the entire results (11 matches, not all of them FP).

Reply:
Indeed. The FPs have been corrected so the rule is ready.

Reply:
Can you test it on the nightly diff corpus using the SentenceSourceChecker, please?

Reply:
I tested it using the SSC, the results seem good.